src/main/resources/openapi/cases.yaml

```diff
@@ -52,6 +52,32 @@ paths:
 
         '500':
           description: Internal Server Error
+    post:
+      tags:
+        - Cases
+      summary: Allows user to add case or if the case exists overwrite the details.
+      description: |-
+        Allows user to add case or if the case exists overwrite the details.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/add_case_request'
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/scheduled_case'
+        '400':
+          description: A required parameter is missing or an invalid datatype or value was provided for property.
+        '409':
+          description: Resource already exists.
```

Comment:
I don't think we would ever return this?

Reply:
Yes, ACCEPT.